### PR TITLE
i476-overview-supported-infra

### PR DESCRIPTION
**Pull request by petra-sargent**
Signed-off-by: Petra Sargent <psargent@redhat.com>

### What does this PR do?

This fixes any grammar and formatting errors for Overview: Supported Infrastructures. This PR contains the infra-support.adoc file that was reviewed and corrected as a part of PR 476 (https://github.com/eclipse/che-docs/pull/476) The old PR 476 will be closed. 

### What issues does this PR fix or reference?
#476

### CODE REVIEW

**Comment by petra-sargent** [comment]
@rkratky Please review. Thank you.

**Comment by petra-sargent** [comment]
@robin-owens Please review. Thanks!

**Comment by petra-sargent** [comment]
@robin-owens @preeticp Please review. Thanks!

**Comment by petra-sargent** [comment]
@rkratky This has been reviewed and I incorporated the feedback. Please merge. Thanks!

File: src/main/pages/overview/infra-support.adoc [hunk 1]
```diff
@@ -10,74 +10,65 @@ folder: overview
...
-== Running Che on Docker
-
-Che on Docker isn’t scalable, i.e. one cannot add more nodes to run workspaces. Of course, it is possible to use a fairly large instance with 8+ CPUs and 32+ RAM, however at some point a great number of running containers (with heavy processes running in them) can make the node unresponsive. This will both affect workspace master and all running workspaces. At the same time, Che on Docker is flexible and easily configurable for an ordinary user.
-
-*Root Access*
+[id="docker-versus-openshift-for-running-che"]
+== Docker versus OpenShift for running Che
 
-With Che on Docker, users can have root access in workspace containers. This means you can run system services and install software in runtime.
+Che on Docker is not scalable because you can not add more nodes to run workspaces. You can use a large instance with 8 CPUs or more and 32+ RAM.  However, a potential problem is that the instance can be unresponsive when it has a large number of running containers with heavy processes running in them. This affects the workspace master and all running workspaces.
```
Comment (rkratky) [inline src/main/pages/overview/infra-support.adoc]: * cannot
* 32+ of what? GB?
* don't use 'however'

File: src/main/pages/overview/infra-support.adoc [hunk 2]
```diff
@@ -10,74 +10,65 @@ folder: overview
...
 
-*SSH Access*
+The follow features are available on both Docker and OpenShift:
 
-By default, `sshd` starts in all ready-to-go stack images. You can connect to a remote workspace using ssh keys or username/password (available in custom stacks only) or sync workspace project files to a local machine.
+*Root Access:* You can have root access in workspace containers, which allows you to run system services and to install software at runtime.
```
Comment (rkratky) [inline src/main/pages/overview/infra-support.adoc:42]: * bullet list?
* unify colon after items

File: src/main/pages/overview/infra-support.adoc [hunk 3]
```diff
@@ -10,74 +10,65 @@ folder: overview
...
 
-*Privileged Containers*
+*Privileged Containers* You can run workspace containers in a link:docker-config.html#privileged-mode[privileged mode].
 
-Che on Docker allows workspace containers to be running in a link:docker-config.html#privileged-mode[privileged mode].
 
-Though deploying administering Che on Docker may seem a little bit easier than doing it in OpenShift, it’s OpenShift that unleashes the power of Eclipse Che as a workspace server and cloud IDE.
+Che on Docker is simpler to deploy but the following features are only available on OpenShift:
```
Comment (rkratky) [inline src/main/pages/overview/infra-support.adoc]: 'Che on Docker is simpler to deploy but' -- redundant

File: src/main/pages/overview/infra-support.adoc [hunk 4]
```diff
@@ -10,74 +10,65 @@ folder: overview
...
 *Persistent Preview URLs*
 
-To access running applications and processes in Che workspaces, Che creates services and routes that are persistent URLs.
+You can access applications and processes in Che workspaces.  Che creates services and routes that are persistent URLs.
 
-*Health Checks*
+*Additional Resources*
 
-OpenShift restarts failed deployments and offers health checks for pods. This can significantly minimize the effect of infrastructure outages.
+*link:docker-single-user[Install Single-User Che on Docker]*
```
Comment (rkratky) [inline src/main/pages/overview/infra-support.adoc]: why bold?

**Review by rkratky — CHANGES_REQUESTED** [review]

File: src/main/pages/overview/infra-support.adoc (hunk 4 above)
Comment (petra-sargent) [reply src/main/pages/overview/infra-support.adoc]: removing the bold

File: src/main/pages/overview/infra-support.adoc (hunk 1 above)
Comment (petra-sargent) [reply src/main/pages/overview/infra-support.adoc]: I will add GB for now but will confirm with @eivantsov. Eugene is it 32 GB of RAM?

**Comment by petra-sargent** [comment]
@rkratky  I incorporated your feedback and tagged Eugene to ask about 32+ of what?

**Comment by petra-sargent** [comment]
@rkratky Please merge. 32 GB of RAM  is okay to use

File: src/main/pages/overview/infra-support.adoc (hunk 1 above)
Comment (petra-sargent) [reply src/main/pages/overview/infra-support.adoc]: Chatted with Eugene on Mattersmost.. Decided to remove "You can use a large instance with 8 CPUs or 32+ RAM"   and replace with "Even if you have a node with lots of resources, you can face problems with many containers residing on one node.

**Review by rkratky — APPROVED** [review]

**Pull request merged**